Automattic/wp-calypso
# Add iframe border and mobile styling

#### Proposed Changes

* This PR adds styling changes for the launchpad.
I have modified the styling for the `Sidebar` and  `WebPreview` components to match the latest Figma designs (9wpoYJrMlYKAKUYvVBMN74-fi-150%3A3809). This includes both desktop and mobile styling.
* I have also added the new props, `showSiteAddressBar` & `devicesToShow` , to the `<WebPreview />` component from https://github.com/Automattic/wp-calypso/pull/66580 to hide the site preview address bar and the tablet icon on the device switcher above the site preview frame.

<img width="1510" alt="image" src="https://user-images.githubusercontent.com/10482592/185424622-083b2e00-79d3-4e3e-b019-a9bcf6d1940b.png">

#### Testing Instructions



* Check out this PR branch
* Navigate to `http://calypso.localhost:3000/setup/launchpad?flow=newsletter&siteSlug=agrullon95.wordpress.com`
* Shrink/Expand the page as needed to view the launchpad page from a mobile & desktop perspective.

#### Pre-merge Checklist






Related to #66571, #66572, #66575
